src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -1173,7 +1173,8 @@ Glyphicon for success status messages
 .alert-success:before {
     font-family: 'Glyphicons Halflings';
     display: inline-block;
-    padding-right: 0.5em;
+    padding-top: .2em;
```

Comment:
It would but what I'm thinking of right now is a little verbose.

See this:
``` CSS
.alert-success:before {
    font-family: 'Glyphicons Halflings';
    display: inline-block;
    padding-top: .15em;
    padding-right: .5em;
    content: '\e084';
    float: left;
}

.alert-success:not('.modal-header'):before {
    font-family: 'Glyphicons Halflings';
    display: inline-block;
    padding-top: 0;
    padding-right: .5em;
    content: '\e084';
    float: left;
}
```